internal/utils.go

```diff
@@ -41,3 +44,29 @@ func CompareTwoValuesInASlice(a reflect.Value, compareFunc func(a, b reflect.Val
 
 	return
 }
+
+func AssertRegexpHelper(t testRunner, regex string, txt string, shouldMatch bool, msg ...interface{}) {
```

Comment:
We should probably accept an interface as regex, since the user could use a string or an already existing regexp. The text might also be an interface, that is later converted to text using `fmt.Sprint()`, so that the user can use any type of input.

Reply:
👍